job_definitions/notify_suppliers_of_framework_application_events.yml

```diff
@@ -49,12 +49,22 @@
             condition: UNSTABLE_OR_WORSE
             predefined-parameters: |
               USERNAME=framework-application-event-email
-              JOB=Notify suppliers of new ${FRAMEWORK_SLUG} framework event {{ environment }}
+              JOB=Notify suppliers of new ${FRAMEWORK_SLUG} framework event - {{ environment }}
               ICON=:frame_with_picture:
               STAGE={{ environment }}
               STATUS=FAILED
               URL=<${BUILD_URL}consoleFull|#${BUILD_NUMBER}>
               CHANNEL=#dm-2ndline
+          - project: notify-slack
+            condition: SUCCESS
+            predefined-parameters: |
+              USERNAME=framework-application-event-email
+              JOB=Notify suppliers of new ${FRAMEWORK_SLUG} framework event - {{ environment }}
```

Comment:
It'd be great if we could include which event it was, e.g. `40_standstill_starts_today`

Reply:
How about adding it as a `PROJECT`?

Reply:
How does that appear in slack?

Reply:
The idea is that it would look something like this, but with the variable filled in

![image](https://user-images.githubusercontent.com/503614/77157001-6cbb2a00-6a98-11ea-9f51-9bb743871d43.png)

Reply:
It looks however like we're limited by whatever variables are defined by the job parameters :(

So either the job needs to be refactored or we have a super long text in the project section.

Reply:
Here's what it looks like with the super long email type

![image](https://user-images.githubusercontent.com/503614/77159933-3da7b700-6a9e-11ea-84f9-187e03342322.png)

Reply:
![image](https://user-images.githubusercontent.com/503614/77175056-ec0d2580-6ab9-11ea-8393-5240f7b626b1.png)

https://gds.slack.com/archives/CAC985MCZ/p1584715643028300